pkg/deploy/util/util.go

```diff
@@ -416,3 +444,16 @@ func (s ByMostRecent) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s ByMostRecent) Less(i, j int) bool {
 	return !s[i].CreationTimestamp.Before(s[j].CreationTimestamp)
 }
+
+// ControllersByCreationTimestampAsc sorts a list of ReplicationControllers by creation timestamp oldest first, using their names as a tie breaker.
+type ControllersByCreationTimestampAsc []api.ReplicationController
```

Comment:
using `ByLatestVersionAsc` ... I expected the list to ascend in versions. Instead, I get this:
```
14467:I0711 12:19:25.585030   14666 controller.go:414] [skuznets] Found 5 prunable deployments, quota is 3
14468:I0711 12:19:25.585043   14666 controller.go:416] [skuznets] Found deployment "history-limit-5" created at {2016-07-11 12:19:12 -0400 EDT}
14469:I0711 12:19:25.585065   14666 controller.go:416] [skuznets] Found deployment "history-limit-4" created at {2016-07-11 12:18:57 -0400 EDT}
14470:I0711 12:19:25.585076   14666 controller.go:416] [skuznets] Found deployment "history-limit-3" created at {2016-07-11 12:18:42 -0400 EDT}
14471:I0711 12:19:25.585088   14666 controller.go:416] [skuznets] Found deployment "history-limit-2" created at {2016-07-11 12:18:28 -0400 EDT}
14472:I0711 12:19:25.585097   14666 controller.go:416] [skuznets] Found deployment "history-limit-1" created at {2016-07-11 12:18:13 -0400 EDT}
```

Am I crazy or is that *de*scending?